src/bp/ml/crf-js/index.ts

```diff
@@ -0,0 +1,77 @@
+import * as sdk from 'botpress/sdk'
+import nanoid from 'nanoid'
+
+import addon, { Tagger as AddonTagger, Trainer as AddonTrainer, Options } from './addon'
+
+export class Trainer implements Omit<AddonTrainer, 'train'>, sdk.MLToolkit.CRF.Trainer {
+  private crfsuite_addon: AddonTrainer
+
+  constructor() {
+    this.crfsuite_addon = new addon.Trainer({ debug: true })
```

Comment:
is debug always set to true ? what are the implication of that ? does it slows down training ?

Reply:
The debug flag was introduced in the new binding... It does not slow down the training, it only enables the callback to be called in train_async.

Basically, it keeps the same behavior as before.